src/layout/BaseLayout/_base-layout.scss

```diff
@@ -79,47 +76,12 @@
     }
   }
 
-  @media (max-width: $breakpoint-medium - 1) {
+  @include mobile {
     &[data-collapsed="true"] {
       transform: translateX(-100%);
     }
   }
 
-  // Override Vanilla to always collapse navigation sidebar
-  @media (min-width: $breakpoint-large - 1) {
```

Comment:
Simplify this by simply setting the Vanilla variable `$application-layout--breakpoint-side-nav-expanded: -1` this way the expand style will never get applied.